src/main/java/com/carmen/app/entities/Car.java

```diff
@@ -25,36 +29,37 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private UUID id;
 	
-	@Column(name = "BRAND_CAR")
+	@Column(name = "brand_car")
 	private String brand;
 	
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "Registration date")
-	private Date registration;
+	//@Temporal(TemporalType.TIMESTAMP)
+	@Column(name ="registration_date")
+	private LocalDateTime registration;
 	
-	@Column(name = "Country")
+	@Column(name = "country")
 	private String country;
 	
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "Created")
-	private Date created_at;
+	//@Temporal(TemporalType.TIMESTAMP)
+	@Column(name = "created")
+	@CreationTimestamp
+	private LocalDateTime created_at;
 	
 	
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "Last Update")
-	private Date last_update;
+	//@Temporal(TemporalType.TIMESTAMP)
+	@Column(name = "last_update")
+	@UpdateTimestamp
```

Comment:
Nice that you've found this annotation. Just to let you know, there are some JPA entities hooks that could also be used here : https://www.baeldung.com/jpa-entity-lifecycle-events#lifecycle